src/main/java/core/basesyntax/dao/FruitDao.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.dao;
+
+public interface FruitDao {
+    void addProduct(String product, int quantity);
+
+    void addProductQuantity(String product, int quantity);
+
+    void subtract(String product, int quantity);
+
+    int getAmount(String product);
+
+    String[] getFruitsNames();
```

Comment:
```suggestion
    int get(String product);

    Map<String, Integer> getAll();
```

Reply:
I need String[] getFruitsNames only for creating report, and from this array I getting fruits names for StringBuilder